op-dispute-mon/metrics/metrics.go

```diff
@@ -312,6 +312,11 @@ func NewMetrics() *Metrics {
 			"delayedWETH",
 			"balance",
 		}),
+		l2Challenges: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Name:      "l2_block_challenges",
+			Help:      "Number of games where the L2 block number has been successfully challenged",
+		}),
```

Comment:
So, while we don't need labels here and to alert on if the l2 block number challenge is against a root we agree with, it'd make the alert and investigation into the issue a lot clearer. Instead of having the investigator look at why the forecasted game result is incorrect through different logs, they'll already see that it's an l2 block number challenge and be able to escalate to proofs and trigger the pre-signed pause. Think that's important enough to warrant the small code addition of an additional alert and labels here.